nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/test/java/org/nuxeo/ecm/platform/audit/io/LogEntrySerializationTest.java

```diff
@@ -0,0 +1,145 @@
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.junit.Test;
+import org.nuxeo.ecm.core.api.impl.blob.StringBlob;
+import org.nuxeo.ecm.platform.audit.api.ExtendedInfo;
+import org.nuxeo.ecm.platform.audit.api.LogEntry;
+import org.nuxeo.ecm.platform.audit.impl.ExtendedInfoImpl;
+import org.nuxeo.ecm.platform.audit.impl.LogEntryImpl;
+
+import java.io.Serializable;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @since 9.3
+ */
+public class LogEntrySerializationTest {
+
+    protected static final String STRING_INFO = "stringInfo";
+
+    protected static final String LONG_INFO = "longInfo";
+
+    protected static final String DOUBLE_INFO = "doubleInfo";
+
+    protected static final String BOOL_INFO = "boolInfo";
+
+    protected static final String DATE_INFO = "dateInfo";
+
+    protected static final String BLOB_INFO = "blobInfo";
+
+    @Test
+    public void testLogEntrySerialization() throws Exception {
+
+        LogEntry entry = new LogEntryImpl();
+        entry.setId(1L);
+
+        Map<String, ExtendedInfo> extendedInfo = new HashMap<>();
+        extendedInfo.put(STRING_INFO, new ExtendedInfoImpl.StringInfo("this is an info"));
+        extendedInfo.put(LONG_INFO, new ExtendedInfoImpl.LongInfo(2L));
+        extendedInfo.put(DOUBLE_INFO, new ExtendedInfoImpl.DoubleInfo(2.0));
+        extendedInfo.put(BOOL_INFO, new ExtendedInfoImpl.BooleanInfo(true));
+
+        String inputString = "21-12-2012";
+        Date date = new SimpleDateFormat("dd-MM-yyyy").parse(inputString);
+        extendedInfo.put(DATE_INFO, new ExtendedInfoImpl.DateInfo(date));
+
+        StringBlob blob = new StringBlob("I'm a blob!");
+        extendedInfo.put(BLOB_INFO, new ExtendedInfoImpl.BlobInfo(blob));
+
+        entry.setExtendedInfos(extendedInfo);
+
+        ObjectMapper mapper = new ObjectMapper();
+        SimpleModule module = new SimpleModule();
+        module.addSerializer(ExtendedInfo.class, new ExtendedInfoSerializer());
+        mapper.registerModule(module);
+
+        String json = mapper.writeValueAsString(entry);
+        assertNotNull(json);
+
+        JsonNode jsonLogEntry = mapper.readTree(json);
+        assertEquals(1L, jsonLogEntry.get("id").asLong());
+
+        JsonNode jsonExtendedInfo = jsonLogEntry.get("extended");
+
+        assertEquals(2L, jsonExtendedInfo.get(LONG_INFO).asLong());
+        assertEquals(2.0, jsonExtendedInfo.get(DOUBLE_INFO).asDouble(), 0);
+        assertEquals("this is an info", jsonExtendedInfo.get(STRING_INFO).asText());
+        assertEquals(true, jsonExtendedInfo.get(BOOL_INFO).asBoolean());
+        assertEquals("2012-12-21T00:00:00Z", jsonExtendedInfo.get(DATE_INFO).asText());
```

Comment:
This is surprising, the input date was in the local timezone if I'm not mistaken, while this is in UTC. There may be a timezone bug.